src/dynacli/bootstrap/_cli.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
I did not understand why do you need this file?

Reply:
This is for entry_point of the package, when you install dynacli from pip - this file's main function acts as entrypoint script.